encoder_node:
  ros__parameters:
    i2c_bus: 1
    home_offset: 3072

controller_node:
  ros__parameters:
    step_pin: 14
    dir_pin: 15
    enable_pin: 18
    base_steps_per_rev: 200
    microstepping: 8
    direction_cw: 1
    direction_ccw: 0
    tolerance_deg: 0.5
    max_speed_delay: 0.0002
    min_speed_delay: 0.002
    deceleration_zone_deg: 20.0
